### PR TITLE
test: Add more tests to CI

### LINKED ISSUE #1129: Add more test cases into CI/CD
Right now in CI/CD, we have ~25% of TCs, and it would be nice to add more cases in order to increase the quality gate.

### PR DESCRIPTION
**Pull request by nalmabrcom**
Fixes #1129

Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

**Comment by nalmabrcom** [comment]
Run in a row multiple times, tests are stable. Log from Jenkins. 
```
10:30:42    (Run Finished)
10:30:42  
10:30:42  
10:30:42         Spec                                              Tests  Passing  Failing  Pending  Skipped  
10:30:42    ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
10:30:42    │ ✔  LSP/copybook.spec.js                     06:16       14       14        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F101836.spec.js                        1ms        -        -        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F102470.spec.js                      00:18        1        1        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F95833.spec.js                       00:41        2        2        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F95834.spec.js                         2ms        -        -        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F96117.spec.js                       00:32        3        3        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F96125.spec.js                       01:33        5        5        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/F97476.spec.js                       00:30        1        1        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/lsp.spec.js                          04:24       13       13        -        -        - │
10:30:42    ├────────────────────────────────────────────────────────────────────────────────────────────────┤
10:30:42    │ ✔  LSP/zowe.spec.js                           1ms        -        -        -        -        - │
10:30:42    └────────────────────────────────────────────────────────────────────────────────────────────────┘
10:30:42      ✔  All specs passed!                        14:16       39       39        -        -        -  
```

File: tests/cypress/integration/LSP/lsp.spec.js [hunk 1]
```diff
@@ -173,6 +173,7 @@ context('This is a LSP spec', () => {
     it(['smoke', 'CI'], 'Checks that Semantic Errors also marked in file', () => {
       cy.openFile('USER2.cbl');
       cy.goToLine(40);
+      console.log();
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/lsp.spec.js:176]: Looks like a debug leftover.

**Review by grianbrcom — APPROVED** [review]
:palm_tree:

**Review by temanbrcom — APPROVED** [review]
🐙

**Pull request merged**